app/src/lib/dispatcher/app-store.ts

```diff
@@ -1682,11 +1682,11 @@ export class AppStore {
 
   /** Set whether the user has opted out of stats reporting. */
   public setStatsOptOut(optOut: boolean): Promise<void> {
-    this.statsStore.setOptOut(optOut)
+    const promise = this.statsStore.setOptOut(optOut)
```

Comment:
could we make this method async and await this call rather than forward the promise since it doesn't look like we return anything out of setOptOut.